mail_tracking_mailgun/models/res_partner.py

```diff
@@ -67,7 +66,7 @@ def check_email_validity(self):
             ):
                 raise UserError(
                     _(
-                        "Error %s trying to check mail" % res.status_code
+                        "Error %s trying to " "check mail" % res.status_code
```

Comment:
Not sure why this change

```suggestion
                        "Error %s trying to check mail" % res.status_code
```